documentation/modules/appendix_crds.adoc

```diff
@@ -2931,6 +2931,8 @@ Used in: xref:type-KafkaRebalance-{context}[`KafkaRebalance`]
 |string array
 |skipHardGoalCheck                        1.2+<.<|Whether to allow the hard goals specified in the Kafka CR to be skipped in rebalance proposal generation. This can be useful when some of those hard goals are preventing a balance solution being found. Default is false.
 |boolean
+|excludedTopics                           1.2+<.<|A regular expression to specify topics that should not be considered for replica movement. This will be parsed by the java.util.regex.Pattern class, so consult the documentation for that class for more details on the supported regex format.
```

Comment:
_Suggested rewording:_

A regular expression that specifies topics to exclude from replica movements in optimization proposals calculated by Cruise Control. `excludedTopics` is parsed by the `java.util.regex.Pattern` class; for more information on the supported regex format, see the documentation for that class.